exchanges/huobi/huobi_test.go

```diff
@@ -1367,6 +1367,40 @@ func TestWSTradeDetail(t *testing.T) {
 	h.SetSaveTradeDataStatus(true)
 	testexch.FixtureToDataHandler(t, "testdata/wsAllTrades.json", h.wsHandleData)
 	close(h.Websocket.DataHandler)
+	exp := []trade.Data{
+		{
+			Exchange:     h.Name,
+			CurrencyPair: btcusdtPair,
+			Timestamp:    time.UnixMilli(1630994963173).UTC(),
+			Price:        52648.62,
+			Amount:       0.006754,
+			Side:         order.Buy,
+			TID:          "102523573486",
+			AssetType:    asset.Spot,
+		},
+		{
+			Exchange:     h.Name,
+			CurrencyPair: btcusdtPair,
+			Timestamp:    time.UnixMilli(1630994963184).UTC(),
+			Price:        52648.73,
+			Amount:       0.006755,
+			Side:         order.Sell,
+			TID:          "102523573487",
+			AssetType:    asset.Spot,
+		},
+	}
+	require.Len(t, h.Websocket.DataHandler, 2, "Must see correct number of trades")
+	for resp := range h.Websocket.DataHandler {
+		switch v := resp.(type) {
+		case trade.Data:
+			i := 1 - len(h.Websocket.DataHandler)
+			require.Equalf(t, exp[i], v, "Trade [%d] must be correct", i)
+		case error:
+			t.Error()
```

Comment:
```suggestion
			t.Error(v)
```
Looks like this was what was intended

Reply:
fixed